azure-pipeline-master.yml

```diff
@@ -1,135 +1,138 @@
-      CONFIG= OSX_FORCE_SDK_DOWNLOAD=1 MACOSX_DEPLOYMENT_TARGET=10.13 run_conda_forge_build_setup
-      conda_forge_ci_setup_activate_sh="${CONDA_PREFIX}/etc/conda/activate.d/conda-forge-ci-setup-activate.sh"
-      sed -Ei.bak "s/^(export MACOSX_DEPLOYMENT_TARGET)=.*$/\\1='10.9'/" "${conda_forge_ci_setup_activate_sh}"
-      rm "${conda_forge_ci_setup_activate_sh}".bak
-      cat "${conda_forge_ci_setup_activate_sh}"
-      grep -qFx "export MACOSX_DEPLOYMENT_TARGET='10.9'" "${conda_forge_ci_setup_activate_sh}"
-      conda list
-    displayName: OSX setup
-
-  - bash: |
-      set -e
-
-      . /usr/local/miniconda/envs/bioconda/etc/profile.d/conda.sh
-      conda activate base
-
-      conda config --system --add channels defaults
-      conda config --system --add channels bioconda
-      conda config --system --add channels conda-forge
-      mkdir -p ${CONDA_PREFIX}/conda-bld
-
-      export OSTYPE="darwin"
-      export CI="true"
-      export HOME=`pwd`
-      bioconda-utils build recipes config.yml \
-          --anaconda-upload \
-          --git-range ${BUILD_SOURCEVERSION}~1 ${BUILD_SOURCEVERSION} \
-          --prelint
-    env:
-      QUAY_OAUTH_TOKEN: $(QUAY_OAUTH_TOKEN)
-      ANACONDA_TOKEN: $(ANACONDA_TOKEN)
-      INVOLUCRO_AUTH: $(INVOLUCRO_AUTH)
-    displayName: Build_and_push
+  - job: "build_and_push_linux"
+    pool:
+      vmImage: "ubuntu-latest"
+    strategy:
+      maxParallel: 8
+    timeoutInMinutes: 360
+
+    steps:
+      - script: |
+          git checkout master
+
+      - bash: echo "##vso[task.prependpath]$CONDA/bin"
+        displayName: Add conda to PATH
+
+      - bash: |
+          wget https://raw.githubusercontent.com/bioconda/bioconda-common/master/common.sh
+        displayName: Fetch common.sh
+
+      - task: Cache@2
+        inputs:
+          path: "/usr/share/miniconda/envs/bioconda"
+          key: '"$(Agent.OS)" | **/common.sh'
+          cacheHitVar: CACHE_RESTORED
+        displayName: Restore cache
+
+      - script: .azure-pipelines/createEnv.py
+        condition: ne(variables.CACHE_RESTORED, 'true')
+        displayName: Install bioconda-utils
+
+      - bash: |
+          set -e
+
+          rm -f "${HOME}/.condarc"
+          # Activate our Conda installation as "base".
+          . /usr/share/miniconda/envs/bioconda/etc/profile.d/conda.sh
+          conda activate base
+
+          conda config --system --add channels defaults
+          conda config --system --add channels bioconda
+          conda config --system --add channels conda-forge
+          docker pull quay.io/dpryan79/mulled_container:latest
+          mkdir -p ${CONDA_PREFIX}/conda-bld
+
+          export OSTYPE="linux-gnu"
+          export CI="true"
+          bioconda-utils handle-merged-pr recipes config.yml \
+            --repo bioconda/bioconda-recipes
+            --git-range ${BUILD_SOURCEVERSION}~1 ${BUILD_SOURCEVERSION}
+            --fallback build
+            --mulled-upload-target biocontainers
```

Comment:
Note that the autoformatter of my editor fixes some non-standard indentation in the YAML files. The actual changes are these here.